consoles/sshVirtsh.pm

```diff
@@ -550,16 +550,16 @@ __END"
     return;
 }
 
-sub attach_to_running ($self, $args) {
-    my $name = ref($args) ? $args->{name} : $args;
+sub attach_to_running ($self, $args = undef) {
+    $args = {name => $args} unless ref $args;
+
+    my $name = $args->{name};
```

Comment:
There are 2 callers of above subroutine in `./tests/installation/bootloader_hyperv.pm` and `./lib/opensusebasetest.pm`. As long as I have not misread it, the hash argument is not used to pass a name. The name parameter is usually part of the object. Would you mind to double-check, please?

Reply:
We *could* change the function to not allow setting the name - at least judging by the uses in "our" test distribution. However, I would like to limit the scope of this PR to fixing the parameter passing and not introduce further changes breaking backwards compatibility.

Reply:
Good point! I haven't thought outside our box. Thanks! :)